cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/Cerberus.java

```diff
@@ -78,10 +77,10 @@ public static void main(String[] args) {
             System.exit(0);
         }
 
-        boolean error = false;
+        Boolean error = false;
         // check required options are present
-        List<String> required = new ArrayList<>();
-        List<String> missing = new ArrayList<>();
+        ArrayList<String> required = new ArrayList<>();
+        ArrayList<String> missing = new ArrayList<>();
```

Comment:
Is the `List` -> `ArrayList` the result of a bad merge?

Reply:
No, as I recall I just used `List` without thinking, and Andre advised that `ArrayList` is better Java practice.

Reply:
I recall suggesting the opposite. The interface is preferred to the concrete class.

Reply:
In that case I'm not sure what `ArrayList` is doing there. Maybe it was a merge error. I'll revert it back to `List`.